### PR TITLE
Add readme and update application parameter

### PR DESCRIPTION
**Pull request by IronPan**
/assign @rmgogogo

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/2021)

### CODE REVIEW

File: manifests/gcp_marketplace/README.md [hunk 1]
```diff
@@ -7,4 +7,41 @@ This directory contains the stacks to deploy on Google Cloud Marketplace.
...
+
+Install application CRD in a cluster
+```
+kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
+```
+
+Install mpdev
+```
+BIN_FILE="$HOME/bin/mpdev"
+docker run  gcr.io/cloud-marketplace-staging/marketplace-k8s-app-tools/k8s/dev:unreleased-pr396  cat /scripts/dev > "$BIN_FILE"
```
Comment (james-jwu) [inline manifests/gcp_marketplace/README.md]: Change double space to single space

File: manifests/gcp_marketplace/README.md [hunk 2]
```diff
@@ -7,4 +7,41 @@ This directory contains the stacks to deploy on Google Cloud Marketplace.
...
+chmod +x "$BIN_FILE"
+export MARKETPLACE_TOOLS_TAG=unreleased-pr396
+export MARKETPLACE_TOOLS_IMAGE=gcr.io/cloud-marketplace-staging/marketplace-k8s-app-tools/k8s/dev
+```
+
+Install Kubeflow Pipelines
+```
+kubectl create ns test
+mpdev /scripts/install  --deployer=gcr.io/ml-pipeline/google/kfp/deployer:0.1.27 --parameters='{"name": "installation-1", "namespace": "test"}'
```
Comment (james-jwu) [inline manifests/gcp_marketplace/README.md]: Double to single space

File: manifests/gcp_marketplace/README.md (hunk 1 above)
Comment (IronPan) [reply manifests/gcp_marketplace/README.md]: done

File: manifests/gcp_marketplace/README.md (hunk 2 above)
Comment (IronPan) [reply manifests/gcp_marketplace/README.md]: done

**Pull request merged**